sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -581,6 +581,13 @@ object SQLConf {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefault(10L)
 
+  val NDV_MAX_ERROR =
+    SQLConfigBuilder("spark.sql.statistics.ndv.maxError")
+      .internal()
+      .doc("The maximum estimation error allowed in HyperLogLog++ algorithm.")
```

Comment:
should we mention that this conf is only used to generate column level statistics?